en/products/relay/matrix.ftl

```diff
@@ -0,0 +1,70 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at https://mozilla.org/MPL/2.0/.
+
+### URL: https://www-dev.allizom.org/products/relay/#pricing
+
+plan-matrix-title = Pricing
+
+## PRICING MATRIX SECTION
+
+plan-matrix-offer-title = Choose a level of protection that’s right for you
+plan-matrix-offer-body = { -brand-name-firefox-relay } protects your identity and your inbox with unique email masks. Add on even more security with { -brand-name-relay } phone masking, and save { $savings } by adding { -brand-name-vpn } protection.
```

Comment:
This doesn't have comments for variables, but I assume the comment for `$savings` above also applies here:

$savings shouldn't include the % symbol here. Different locales have different ways of styling numbers/percentages, however we are forcing those locales to use the English styling/typography (70% versus 70 % for French). The % should be part of the Fluent string so locales can style appropriately: e.g. Add it to { -brand-name-firefox-relay } for { $savings }% off.